### PR TITLE
#1284 avs tikl update

### PR DESCRIPTION
**Pull request by MDR22**
Updated create_TIKL to 11 days instead of 10 to be consistent with ONEsource manual to initiate a case review 10 calendar days AFTER an ad-hoc request was submitted in AVS.

### CODE REVIEW

File: notes/avs.vbs [hunk 1]
```diff
@@ -556,10 +557,10 @@ Do
     End if
 
     'Call create_TIKLL(TIKL_text, num_of_days, date_to_start, ten_day_adjust, TIKL_note_text)
-    If set_form_TIKL = True then Call create_TIKL("DHS-7823 - AVS Auth Form(s) have been requested for this case. Review case file/notes, and take applicable actions.", 10, date, False, TIKL_note_text)
-    If set_AVS_TIKL = True then Call create_TIKL("AVS 10-day check is due.", 10, date, False, TIKL_note_text)
-    If set_another_TIKL = True then Call create_TIKL("An updated DHS-7823 - AVS Auth Form(s) has been requested for this case. Review case file/notes, and take applicable actions.", 10, date, False, TIKL_note_text)
-    If set_asset_TIKL = True then Call create_TIKL("AVS unreported asset verification requested for the case. Review case file/notes, and take applicable actions.", 10, date, False, TIKL_note_text)
+    If set_form_TIKL = True then Call create_TIKL("DHS-7823 - AVS Auth Form(s) have been requested for this case. Review case file/notes, and take applicable actions.", 11, date, False, TIKL_note_text)
```
Comment (C-Love) [inline notes/avs.vbs:560]: This is a verification request and would stay at 10 days.

File: notes/avs.vbs [hunk 2]
```diff
@@ -556,10 +557,10 @@ Do
...
+    If set_AVS_TIKL = True then Call create_TIKL("AVS 10-day check is due.", 11, date, False, TIKL_note_text)
+    If set_another_TIKL = True then Call create_TIKL("An updated DHS-7823 - AVS Auth Form(s) has been requested for this case. Review case file/notes, and take applicable actions.", 11, date, False, TIKL_note_text)
```
Comment (C-Love) [inline notes/avs.vbs:562]: This is a verification request and would stay at 10 days.

File: notes/avs.vbs [hunk 3]
```diff
@@ -556,10 +557,10 @@ Do
...
+    If set_asset_TIKL = True then Call create_TIKL("AVS unreported asset verification requested for the case. Review case file/notes, and take applicable actions.", 11, date, False, TIKL_note_text)
```
Comment (C-Love) [inline notes/avs.vbs:563]: This is a verification request and would stay at 10 days.

File: notes/avs.vbs [hunk 4]
```diff
@@ -44,6 +44,7 @@ changelog = array()
 
 'INSERT ACTUAL CHANGES HERE, WITH PARAMETERS DATE, DESCRIPTION, AND SCRIPTWRITER. **ENSURE THE MOST RECENT CHANGE GOES ON TOP!!**
 'Example: call changelog_update("01/01/2000", "The script has been updated to fix a typo on the initial dialog.", "Jane Public, Oak County")
+call changelog_update("05/01/2023", "Updated TIKL to be set for 11 days instead of 10 to align with timeline in ONEsource manual for AVS.", "Mark Riegel, Hennepin County")
```
Comment (C-Love) [inline notes/avs.vbs:47]: Clarify that this 11 day TIKL is for the AVS Portal Case Review reminder after submitting an Ad-Hoc request.

**Review by C-Love — CHANGES_REQUESTED** [review]
The change to 11 days out for the TIKL is only for the AVS action to check the submission. Any that are a return of verifications would still be 10 days.

File: notes/avs.vbs (hunk 1 above)
Comment (MDR22) [reply notes/avs.vbs]: Changed back to 10 days

File: notes/avs.vbs (hunk 2 above)
Comment (MDR22) [reply notes/avs.vbs]: Changed back to 10 days

File: notes/avs.vbs (hunk 3 above)
Comment (MDR22) [reply notes/avs.vbs]: Changed back to 10 days

File: notes/avs.vbs (hunk 4 above)
Comment (MDR22) [reply notes/avs.vbs]: Updated changelog language to clarify

**Review by C-Love — APPROVED** [review]
Looks Great!

**Pull request merged**